### PR TITLE
[CALCITE-6543] Change the RelOptCostImpl toString method to be consistent with VolcanoCost

### PR DESCRIPTION
**Pull request by ChengJie1053**
image

### CODE REVIEW

**Comment by caicancai** [comment]
I think this PR deserves a jira case

**Comment by ChengJie1053** [comment]
> I think this PR deserves a jira case

Ok, thanks for reviewing the code for me

**Comment by ChengJie1053** [comment]
> Details
Hello, jira has been created

https://issues.apache.org/jira/projects/CALCITE/issues/CALCITE-6543?filter=allissues

**Comment by ChengJie1053** [comment]
> I think this PR deserves a jira case

Hello, jira has been created
https://issues.apache.org/jira/projects/CALCITE/issues/CALCITE-6543?filter=allissues

File: core/src/main/java/org/apache/calcite/plan/RelOptCostImpl.java [hunk 1]
```diff
@@ -115,6 +115,12 @@ public RelOptCostImpl(double value) {
   @Override public String toString() {
     if (value == Double.MAX_VALUE) {
       return "huge";
+    } else if (value == Double.POSITIVE_INFINITY) {
```
[the remark this answers is not in the document]
Comment (ChengJie1053) [reply core/src/main/java/org/apache/calcite/plan/RelOptCostImpl.java:118]: Ok, thanks for reviewing the code for me

**Comment by mihaibudiu** [comment]
Ideally you should avoid code duplication, especially when the control-flow is so complicated.
In this case you should probably create a static method that takes a double and returns a string, and call it from both places.

**Comment by ChengJie1053** [comment]
> Ideally you should avoid code duplication, especially when the control-flow is so complicated. In this case you should probably create a static method that takes a double and returns a string, and call it from both places.

Ok, thank you for helping me review the code, I will modify this place

**Comment by ChengJie1053** [comment]
> Ideally you should avoid code duplication, especially when the control-flow is so complicated. In this case you should probably create a static method that takes a double and returns a string, and call it from both places.

Hello, the modified code has been submitted. If there is any problem, I will modify it again

**Comment by ChengJie1053** [comment]
> squash

Ok, thank you. I mistakenly thought that every commit requires a squash commit. I will pay attention to that in the future

**Pull request merged**